src/components/AdviceGeneratorApp/AdviceGeneratorApp.tsx

```diff
@@ -1,15 +1,20 @@
-import { getAdvices } from "../../helpers/getAdvices"
+// import { getAdvices } from "../../helpers/getAdvices"
+import { useFetchAdvices } from "../../helpers/useFetchAdvices"
 
 export const AdviceGeneratorApp = () => {
 
-  getAdvices();
+  const { adviceState } = useFetchAdvices();
 
```

Comment:
Con base a las propiedades propuestas, puedes hacer un render condicional:

```suggestion
    if(loading) // O alternativamente puedes hacer if(!advice)
      return <h1>Loading...</h1>
    
   if(error)
      return <h1>There was an error at fetching the advice</h1>
```